internal/operators/manager.go

```diff
@@ -95,19 +100,16 @@ func (mgr *Manager) ValidateHost(ctx context.Context, cluster *common.Cluster, h
 
 	results := make([]api.ValidationResult, 0, len(mgr.operators))
 
-	// To track operators that are disabled or not present in the cluster configuration, but have to be present
-	// in the validation results and marked as valid.
-	pendingOperators := make(map[models.OperatorType]bool)
+	pendingOperators := make(map[models.OperatorType]struct{})
```

Comment:
Why `struct{}` in favor of a boolean? Boolean values in map-based sets are more practical, as checking for presence of a key is simpler.

Reply:
with `struct{}` you're allocating 0 bytes, whereas with a bool you're allocating 1 byte ([goplayground](https://play.golang.org/p/T6plnXgQgaH)). Since you're not using the value of the map for this loop (only the key to identify which operators are pending), it sounded reasonable to change it to `struct{}` to avoid throwing the byte away.

Let me know if you still want me to revert it back to bool.

Reply:
OK, I'm fine with your change.